common/lib/xmodule/xmodule/templates/problem/jsinput_response.yaml

```diff
@@ -161,7 +161,7 @@ data: |
       }
       </script>
         <p>What is the range of passengers that can fit on a 747 aircraft? Set the low and high bounds. (416 and 660 persons)</p>
-        <p class="sr" aria-hidden="true">
+        <p class="is-hidden" aria-hidden="true">
```

Comment:
I thought this text was part of what made this problem accessible. What actually makes this problem accessible?